api/database/migrations/2022_10_06_220924_replace_pool_publish_with_time.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class ReplacePoolPublishWithTime extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('pools', function (Blueprint $table) {
+            $table->timestamp('published_at')->nullable();
+        });
+        DB::statement("
+            update pools
+                set published_at =
+                    case is_published
+                        when 'true' then TIMESTAMP '2022-10-01 00:00:00'
```

Comment:
oh yeah, that was one thing I had a question about, how do you appropriately pass a variable into a DB Statement?

Reply:
I'm not sure I've ever done it in a migration but it probably looks something like this:
https://laravel.com/docs/8.x/database#using-named-bindings
But probably a better way to do it is to use a current date/time function instead of passing in a parameter:
https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT

Reply:
updated with https://github.com/GCTC-NTGC/gc-digital-talent/pull/4244/commits/030b0d2be9d0b921fff42c0c2a4fb3fa0b31578d